plugin/lables.go

```diff
@@ -0,0 +1,93 @@
+package plugin
```

Comment:
You should update this file name to be labels.go rather than lables.go

Reply:
ok